src/edu/washington/escience/myria/util/DeploymentUtils.java

```diff
@@ -205,8 +216,8 @@ public static void startWorker(final String address, final String workingDir, fi
     builder.append(" edu.washington.escience.myria.parallel.Worker");
     builder.append(" --workingDir " + workerDir);
     builder.append(" 0</dev/null");
-    builder.append(" 1>" + path + "/worker_" + workerId + "_stdout");
-    builder.append(" 2>" + path + "/worker_" + workerId + "_stderr");
+    builder.append(" 1>" + workerDir + "/worker_" + workerId + "_stdout");
```

Comment:
Again I think this could be simplified by not duplicating `workerID` in filenames.